app/models/benefitting_country.rb

```diff
@@ -0,0 +1,40 @@
+class BenefittingCountry
```

Comment:
I really like the approach of having a class for this entity!

Reply:
Thanks! 👍